admin/config-ui/class-configuration-page.php

```diff
@@ -104,8 +104,7 @@ public function show_wizard() {
 				printf(
 					/* translators: %s expands to Yoast SEO. */
 					__( '%s &rsaquo; Configuration Wizard', 'wordpress-seo' ),
-					'Yoast SEO'
-				);
+					'Yoast SEO' );
```

Comment:
I disagree that this is a (better) solution, this will create spaces and newlines in the actual HTML output, which can result in unwanted visual changes.